projects/gfbio-app/src/app/oidc/oidc.component.ts

```diff
@@ -0,0 +1,68 @@
+import {Component, OnInit, Inject, OnDestroy} from '@angular/core';
+import {UserService, Config, User} from "wave-core";
+import {AppConfig} from "../app-config.service";
+import {first, SubscriptionLike} from "rxjs";
+
+@Component({
+    selector: 'wave-gfbio-login',
+    templateUrl: 'oidc.component.html',
+    styleUrls: ['./oidc.component.scss'],
+})
+export class OidcComponent implements OnInit, OnDestroy {
+    private userSubscription : SubscriptionLike | undefined;
+    private loginSubscription : SubscriptionLike | undefined;
+    private logoutSubscription : SubscriptionLike | undefined;
+
+    user? : User;
+
+    constructor(
+        @Inject(Config) private readonly config: AppConfig,
+        private readonly userService: UserService,) {
+    }
+
+    ngOnInit(): void {
+        this.userSubscription = this.userService
+            .getSessionStream()
+            .pipe(first())
+            .subscribe((session) => {
+                if (!session.user || session.user.isGuest) {
+                } else {
+                    this.user = session.user;
+                }
```

Comment:
It is either missing something, or you mean:

```suggestion
                if (!session.user || session.user.isGuest) {
                    return;
                }
                
                this.user = session.user;
```